cookbook/extend-entities.rst

```diff
@@ -94,6 +94,11 @@ For the `User` entity (`se_users`):
                 model: App\Entity\User
                 repository: Sulu\Bundle\SecurityBundle\Entity\UserRepository
 
+.. note::
+
+   Symfony keeps the user object in the session, clearing the sessions is so sometimes required
```

Comment:
add a link to the symfony command how to clear the sessions

Reply:
@wachterjohannes dont know any command which clears the sessions.

Reply:
how have you done it then in the project we had that problem?

Reply:
Delete the session files manual they are stired differently based on the OS